application-analytics-api/src/main/java/com/xwiki/analytics/configuration/AnalyticsConfiguration.java

```diff
@@ -51,4 +51,25 @@ public interface AnalyticsConfiguration
      * @return the tracking code for Matomo.
      */
     String getTrackingCode();
+
+    /**
+     * @return the update interval.
```

Comment:
I think this parameter method should be more specific because the getInterval is specific to the lastVisit macro. I will update the method name to -> lastSeenUpdateInterval